src/libpriv/rpmostree-scripts.c

```diff
@@ -158,7 +158,25 @@ static const RpmOstreeLuaReplacement lua_replacements[] = {
     "/usr/bin/sh",
     "set -euo pipefail\n"
     "echo %{_install_langs} >/usr/share/rpmostree-lua-override-test-expand\n"
-  },
+  }
+};
+
+typedef struct {
+  const char *pkgname_script;
+  const char *release_suffix;
+  const char *interp;
+  const char *replacement;
+} RpmOstreeScriptReplacement;
+
+static const RpmOstreeScriptReplacement script_replacements[] = {
+  /* Only neuter the rhel7 version; the Fedora one is fixed.
+   * https://src.fedoraproject.org/rpms/pam/pull-request/3
+   */
+  { "pam.post", ".el7", NULL, NULL },
+  /* Same here. */
+  { "microcode_ctl.post", ".el7", NULL, NULL },
+  /* And this one runs dracut, which we don't want. */
+  { "microcode_ctl.posttrans", ".el7", NULL, NULL },
```

Comment:
Can't we make this part of the gperf lookup table? E.g. add a new `release_suffix` entry there? Just to try to keep lookups and lists centralized.

Reply:
Right, we'd need to add a new field like `release_suffix` to the table to make sure it doesn't get subbed out on Fedora (I haven't used `gperf` much but looking at https://www.gnu.org/software/gperf/manual/gperf.html#User_002dsupplied-Struct it seems like that should be doable).